File: Practica7/Juego2/Java/Juego.java
class Juego {
    int nroVida;
    int record;
    public void reiniciaPartida() {
        nroVida = 3;
    }
    public void actualizaRecord() {
        record++;
    }
    public boolean quitaVida() {
        nroVida--;
        return nroVida > 0;
    }
}

File: Practica7/Juego2/Python/Juego.py
class Juego:
    def __init__(self):
        self.nroVida = 3
        self.record = 0
    def reiniciaPartida(self):
        self.nroVida = 3
    def actualizaRecord(self):
        self.record += 1
    def quitaVida(self):
        self.nroVida -= 1
        return self.nroVida > 0